views/index.pug

```diff
@@ -3,6 +3,36 @@ extends layout
 block content
   h1= title
   p Welcome to #{title}
+
+  span#start.target
+
+  span#about.target
+
+  .modal
+    .content.vertical-align-middle
```

Comment:
The class names you are using describe the positioning of elements, rather than their responsibility.  What happens when you want to move this `content` to the left of the page?  Do you change the class name? Leave the class name and update the styles, breaking the semantic connection?

I recommend removing classes that describe the position, and use classes that describe the content.  For this specific case, you could simply omit the `vertical-align-middle`, and re-apply those styles to `.modal .content`